task-bht_bold.psyexp

```diff
@@ -812,10 +727,10 @@
         <Param val="" valType="extendedCode" updates="constant" name="Begin Experiment"/>
         <Param val="" valType="extendedCode" updates="constant" name="Begin JS Experiment"/>
         <Param val="/* Syntax Error: Fix Python code */" valType="extendedCode" updates="constant" name="Begin JS Routine"/>
-        <Param val="send_message(b&quot;\x02&quot;)" valType="extendedCode" updates="constant" name="Begin Routine"/>
+        <Param val="send_message(b&quot;\x02&quot;)&amp;#10;ioServer.getDevice('tracker').sendMessage(&quot;bye bht&quot;)" valType="extendedCode" updates="constant" name="Begin Routine"/>
         <Param val="Auto-&gt;JS" valType="str" updates="None" name="Code Type"/>
-        <Param val="" valType="extendedCode" updates="constant" name="Each Frame"/>
-        <Param val="" valType="extendedCode" updates="constant" name="Each JS Frame"/>
+        <Param val="if etRecord_2.status == NOT_STARTED:&amp;#10;    etRecord_2.status = STARTED" valType="extendedCode" updates="constant" name="Each Frame"/>
```

Comment:
I included this line because the code generated by Psychopy sets the value to "NOT_STARTED" at the beginning of the routine. As a result, the code never entered the loop that turned off the eye-tracker.

Reply:
Outstanding! Thanks for catching this. Did you test this with the ET yesterday?

Reply:
Yes, I tested it out with the ET yesterday, and now it's working!